packages/host/app/services/matrix-service.ts

```diff
@@ -393,6 +397,13 @@ export default class MatrixService extends Service {
     }
   }
 
+  private addTools = (attachedOpenCard: CardDef, patchSpec: any) => {
+    return [
+      getPatchTool(attachedOpenCard, patchSpec),
+      getSearchTool(attachedOpenCard),
+    ];
```

Comment:
same comment as above around figuring out a better way to encapsulate command specific logic.

Reply:
I think having these around are ok for now since its in the host, not the ai bot. They are just const function tool calls that gets attached to a message. Can we wait for the command card to exist before thinking of removing this.